device_backends/uio/src/UioBackend.cc

```diff
@@ -6,7 +6,7 @@
 namespace ChimeraTK {
 
   UioBackend::UioBackend(std::string deviceName, std::string mapFileName) : NumericAddressedBackend(mapFileName) {
-    _uioAccess = std::shared_ptr<UioAccess>(new UioAccess("/dev/" + deviceName));
+    _uioAccess = std::make_shared<UioAccess>("/dev/" + deviceName);
```

Comment:
do these changes really belong into the same commit with the other changes?

Reply:
Well, they are minor improvements in the noise of the changes which are necessary in that file anyway. I did not consider them worth a separate commit.